client/ayon_hiero/plugins/publish/collect_workfile.py

```diff
@@ -0,0 +1,21 @@
+import pyblish.api
+
+import hiero
+
+
+class CollectWorkfile(pyblish.api.InstancePlugin):
+    """Collect the current working file into context"""
+
+    label = "Collect Workfile"
+    families = ["workfile"]
```

Comment:
```suggestion
```

The `currentFile` should ALWAYS be injected into the `context` - even if `workfile` instance is inactive. So no family is needed.

Also, this can be a `ContextPlugin` so it runs only once instead of per instance.

Reply:
Good call, thanks for this suggestion !